listenbrainz/webserver/templates/user/base.html

```diff
@@ -4,7 +4,7 @@
 
 {% block content %}
   <div id="user-profile">
-    <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
+    <div id="follow-controls-container"></div>
```

Comment:
Should maybe be `user-page-heading-container`?

Reply:
Makes total sense. done.